corehq/apps/reports/models.py

```diff
@@ -240,3 +248,251 @@ class TableauUser(models.Model):
     groups = models.ManyToManyField(TableauGroup)
     tableau_user_id = models.CharField(max_length=64)
     last_synced = models.DateTimeField(auto_now=True)
+
+
+logger = logging.getLogger('tableau_api')
+logger.setLevel('DEBUG')
```

Comment:
This shouldn't be done here. The logging config is done in settings and localsettings e.g. https://github.com/dimagi/commcare-hq/blob/master/settings.py#L1470-L1474